website/source/docs/virtualbox/networking.html.md

```diff
@@ -10,6 +10,22 @@ description: |-
 
 # Networking
 
+## VirtualBox Host-Only Networks
+
+By default, private networks are [host-only networks](https://www.virtualbox.org/manual/ch06.html#network_hostonly),
+because those are the easiest to work with. In VirtualBox, since you can
+create multiple host-only networks, it is also possible to specify which
+host-only network you want the Vagrant VirtualBox provider to use for
+a given interface. To do this, use the `name` argument with the name of
+the host-only interface to use.
+
+```ruby
+Vagrant.configure("2") do |config|
+  config.vm.network "private_network", type: "dhcp",
+    name: => "vboxnet3"
```

Comment:
Ahhh shoot! Sorry, this should just be `name: "vboxnet3"` :)

Reply:
Oh gosh, I'm sorry. Of course. Fixed now.